### PR TITLE
Update Compiler-Options-Hardening-Guide-for-C-and-C++.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
added wg contributors

### CODE REVIEW

**Review by thomasnyman — APPROVED** [review]

**Comment by thomasnyman** [comment]
Should we reach out to these individuals to confirm they consent to be named in the guide? I thought about adding everyone as reviewers to this PR, but apparently that's only possible for those who are part of the org :(

**Review by david-a-wheeler — APPROVED** [review]

File: docs/Compiler_Hardening_Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md [hunk 1]
```diff
@@ -709,10 +709,20 @@ If the build ID method is used the debug info file’s name is computed from the
...
 - Thomas Nyman, Ericsson
 - Robert Byrne, Ericsson
 - Jussi Auvinen, Ericsson
+- Christopher "CRob" Robinson, Intel
+- David A. Wheeler, Linux Foundation
+- Gabriel Dos Reis, Microsoft
+- Georg Kunz, Ericsson
+- Jack Kelly, ControlPlane
+- Randall T. Vasquez, Linux Foundation
+- Robert C. Seacord, Woven Planet
```
Comment (rcseacord) [inline docs/Compiler_Hardening_Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md:723]:
```suggestion
- Robert C. Seacord, Woven by Toyota
```

**Review by rcseacord — APPROVED** [review]

**Comment by thomasnyman** [comment]
So far CRob, Jack, Siddharth, and Siddhesh  have communicated their approval via e-mail, in addition to those who have already approved this PR here.

**Review by 06kellyjac — APPROVED** [review]

**Review by ran-dall — APPROVED** [review]

**Review by GabrielDosReis — APPROVED** [review]

**Comment by SecurityCRob** [comment]
i will create a new pr with 100% less conflicts updating this same information

**Comment by david-a-wheeler** [comment]
You're certainly welcome to refer to me :-).

**Review by david-a-wheeler — APPROVED** [review]

**Comment by thomasnyman** [comment]
Closing this PR as these changes have been merged via #224.

**Comment by thomasnyman** [comment]
Closing this PR as these changes have been merged via #224.

**Pull request closed without merging**